### ISSUE TITLE
Question: Parallel with serial

### ISSUE THREAD

**Issue by amilajack** [mysticatea/npm-run-all#58]
Is it possible to compile serial and parallel tasks? Ex:

```bash
run-s lint run-p build
```

**Comment by mysticatea** [comment]
Thank you for this issue.

Though I might not understand this issue, I think there are some solutions:

```
npm-run-all -s lint -p build:*
```

```
npm run lint && run-p build:*
```

```
"prebuild": "npm run lint",
"build": "run-p build:*"
```

**Comment by evilebottnawi** [comment]
@mysticatea i think @amilajack meant nested task, example:
                 /- `favicons` -> `imagemin` (`serial`)
                /-- `webfont` (`serial`)
`parallels`
                \-- `copy` (`serial`)
                 \-`another-task` -> `another-task-too` -> `another-task-too-too` (`serial`)

**Comment by kentor** [comment]
Yeah.. nested tasks within a parallel block would be useful. i guess it's possible if you added another script for the nested block but that doesn't scale well
